pkgs/by-name/fl/fluent-bit/package.nix

```
@@ -1,83 +1,145 @@
 {
   lib,
-  stdenv,
-  fetchFromGitHub,
+  bison,
+  c-ares,
   cmake,
+  curl,
+  fetchFromGitHub,
   flex,
-  bison,
-  systemd,
-  postgresql,
-  openssl,
+  jemalloc,
+  libbacktrace,
+  libbpf,
+  libnghttp2,
   libyaml,
-  darwin,
+  luajit,
+  nix-update-script,
+  nixosTests,
+  openssl,
+  pkg-config,
+  postgresql,
+  rdkafka,
+  stdenv,
+  systemd,
+  versionCheckHook,
+  zlib,
+  zstd,
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation rec {
   pname = "fluent-bit";
   version = "3.2.2";
 
   src = fetchFromGitHub {
     owner = "fluent";
     repo = "fluent-bit";
-    rev = "v${finalAttrs.version}";
+    rev = "refs/tags/v${version}";
```

```suggestion
    tag = "v${version}";
```

`fetchFromGitHub` gained a new `tag` parameter. :)